Modules/_ctypes/stgdict.c

```diff
@@ -350,11 +350,12 @@ PyCStructUnionType_update_stgdict(PyObject *type, PyObject *fields, int isStruct
     _Py_IDENTIFIER(_swappedbytes_);
     _Py_IDENTIFIER(_use_broken_old_ctypes_structure_semantics_);
     _Py_IDENTIFIER(_pack_);
+    _Py_IDENTIFIER(_ms_struct_);
```

Comment:
At the moment, this is just so that we can test `msvc` compatibility mode on Linux and vice versa.

We can expose it to users via the docs, too, if we want to.